<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcDialog</class>
 <widget class="QDialog" name="CalcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,3">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="hexRadioButton">
       <property name="text">
        <string>HEX</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="decRadioButton">
       <property name="text">
        <string>DEC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="binRadioButton">
       <property name="text">
        <string>BIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="keyboardsLayout">
     <item>
      <widget class="QWidget" name="decKeyboard" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="decORButton">
            <property name="text">
             <string>OR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decANDButton">
            <property name="text">
             <string>AND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec7Button">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec4Button">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec1Button">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec0Button">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="decXORButton">
            <property name="text">
             <string>XOR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decClearButton">
            <property name="text">
             <string>CLEAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec8Button">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec5Button">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec2Button">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decSubButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="decCMPButton">
            <property name="text">
             <string>~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decDelButton">
            <property name="text">
             <string>DEL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec9Button">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec6Button">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dec3Button">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decEqButton">
            <property name="text">
             <string>=</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="decMODButton">
            <property name="text">
             <string>MOD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decSRButton">
            <property name="text">
             <string>SHIFT -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decSLButton">
            <property name="text">
             <string>SHIFT &lt;-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decMulButton">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decDivButton">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="decAddButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="hexKeyboard" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="hexANDButton">
            <property name="text">
             <string>AND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexEButton">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexDButton">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexCButton">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexBButton">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexAButton">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="hexORButton">
            <property name="text">
             <string>OR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexFButton">
            <property name="text">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex7Button">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex4Button">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex1Button">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex0Button">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="hexXORButton">
            <property name="text">
             <string>XOR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexClearButton">
            <property name="text">
             <string>CLEAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex8Button">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex5Button">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex2Button">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexSubButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QPushButton" name="hexCMPButton">
            <property name="text">
             <string>~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexDelButton">
            <property name="text">
             <string>DEL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex9Button">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex6Button">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hex3Button">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexEqButton">
            <property name="text">
             <string>=</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QPushButton" name="hexMODButton">
            <property name="text">
             <string>MOD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexSRButton">
            <property name="text">
             <string>SHIFT -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexSLButton">
            <property name="text">
             <string>SHIFT &lt;-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexMulButton">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexDivButton">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hexAddButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="binKeyboard" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QPushButton" name="binClearButton">
            <property name="text">
             <string>CLEAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binDelButton">
            <property name="text">
             <string>DEL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bin1Button">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bin0Button">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binSubButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binMulButton">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binANDButton">
            <property name="text">
             <string>AND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binXORButton">
            <property name="text">
             <string>XOR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QPushButton" name="binSRButton">
            <property name="text">
             <string>SHIFT -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binSLButton">
            <property name="text">
             <string>SHIFT &lt;-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binAddButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binDivButton">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binEqButton">
            <property name="text">
             <string>=</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binCMPButton">
            <property name="text">
             <string>~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binORButton">
            <property name="text">
             <string>OR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="binMODButton">
            <property name="text">
             <string>MOD</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
